packages/starknet/lib/src/account.dart

```diff
@@ -103,6 +103,15 @@ class Account {
 
     return getDeployedContractAddress(txReceipt);
   }
+
+  Future<Uint256> balance() async =>
+      ERC20(account: this, address: ethAddress).balanceOf(accountAddress);
+
+  Future<bool> send({required Felt recipient, required Uint256 amount}) async {
```

Comment:
I'm not sure if it's better to return **success** of transaction or the transaction hash

Reply:
I would  send the the tx hash and have the user decide if she wants to wait or not (ux concern)

Reply:
PR updated to return **tx hash** instead of **success**